hw/ip/otbn/doc/_index.md

```diff
@@ -707,7 +707,7 @@ Detected integrity violations in the data memory raise a fatal `imem_error`.
 
 ### Secure Wipe {#design-details-secure-wipe}
 
-The entire OTBN state, including the contents of instruction and data memories, can be securely deleted on demand from a host software.
+The contents of instruction and data memories can be securely deleted on demand from a host software.
 In addition, full or partial secure wipe is triggered automatically by the OTBN in certain situations.
```

Comment:
That sentence is now missing the internal wipe. Maybe reword the two sentences completely to first describe what types of secure wipe we have, and then how it can be triggered (roughly, it's the intro!)